lib/puppet/provider/package/cisco.rb

```diff
@@ -30,7 +31,7 @@
   # these commands must exist to execute native yum provider
   commands yum: 'yum', rpm: 'rpm', python: 'python'
 
-  defaultfor operatingsystem: :nexus
+  defaultfor operatingsystem: [:ios_xr, :nexus]
```

Comment:
Reading puppet type and provider book, it seems we do not need to set "defaultfor" at all.